solutions/USACO feb16-milk-pails.mdx

```diff
@@ -53,4 +53,56 @@ int main () {
 ```
 
 </CPPSection>
+<JavaSection>
+
+## Java Implementation
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class pails {
+    static int x, y, k, m, sol;
+    static boolean[][][] vis;
+    public static void main(String[] args) throws IOException {
+        Scanner sc = new Scanner(new File("pails.in"));
+        PrintWriter out = new PrintWriter("pails.out");
+
+        x = sc.nextInt();
+        y = sc.nextInt();
+        k = sc.nextInt();
+        m = sc.nextInt();
+        sol = m;
+
+        vis = new boolean[x+1][y+1][k+1];
+        dfs(0, 0, 0);
+        
+        out.println(Math.abs(sol));
```

Comment:
Don't need Math.abs here right?

Reply:
Yep, my mistake!

Reply:
Not a mistake, per se, but in general it's preferable to have simpler code.